en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+
+You can use the admin API that is available in WSO2 API Manager to publish the summarized data to Stripe. After this API call takes place, it pushes the usage data to Stripe. After Stripe gets the usage data, it will check for the subscriptions that are completing their billing cycle and charge the customer accordingly.
+
+1.  Obtain the consumer key and secret key pair by calling the dynamic client registration endpoint.  
+     
+     For more information, see [Admin REST API - Getting Started](https://docs.wso2.com/display/AM260/apidocs/admin/index.html#guide).
+
+    ``` java
+    curl -X POST -H "Authorization: Basic <base64encoded-admin-account-credentials>" -H "Content-Type: application/json" -d @payload.json https://localhost:9443/client-registration/v0.14/register
+    ```
+
+    -   `<base64encoded-admin-account-credentials>` - Use a base64 encoder (e.g., <https://www.base64encode.org/> ) to encode the username and password       that corresponds to your admin user account using the following format: 
+        
+        `<username>:<password>` 
+        
+        Thereafter, enter the encoded value as this parameter.
+
+2.  Obtain a token with the monetization usage scope (`scope apim:monetization_usage_publish`).
+
+    ``` java
+    curl -X POST https://localhost:8243/token -H 'Authorization: Basic <base64encoded-registeration-credentials>' -d 'grant_type=password&username=admin&password=admin&scope=apim:monetization_usage_publish'
+    ```
+      
+    -   `<base64encoded-registeration-credentials>` - Use a base64 encoder (e.g., <https://www.base64encode.org/>) to encode the username and password          that corresponds to the client ID and client secret that you received as the response in the previous step using the following format: 
+    
+        `<clientID>:<clientSecret>` 
+    
+        Thereafter, enter the encoded value as this parameter.
+
+3.  Publish usage data to the Stripe billing engine.
+
+    ``` java
+    curl -k -H "Authorization: Bearer <monetization-usage-publish-token>" -X POST -H "Content-Type: application/json" https://localhost:9443/api/am/admin/v0.14/monetization/publish-usage
+    ```
+
+    -   `<monetization-usage-publish-token>` - This is the token that you got when you executed the Admin REST API with the monetization usage scope.
+
+    The REST API response is as follows:
+
+    ``` java
+    {"status":"Request Accepted","message":"Server is running the usage publisher"}
+    ```
+
+    The aggregation count will be stored in `ApiPerDestinationAgg_SECONDS` table, which corresponds to the WSO2 API-M Analytics database. If you check the `AM_MONETIZATION_USAGE_PUBLISHER` table after the admin API call is made, you will see that the record for the API usage is published.  
+      
+    In addition, after the admin API call is made the aggregated bill
+    gets generated in Stripe (connected account).  
+    ![Pricing](../../assets/img/Learn/pricing.png) At the end of
```

Comment:
same here

Reply:
Fixed.
I added the path based on the example on page 6 in the "Guidelines for Github Documentation" document. Looks like I misinterpreted the example.